include/pp_profile.hh

```diff
@@ -104,8 +104,43 @@ struct ProfileMap : private map < pair<int, int>, double > {
     }
 }; // class ProfileMap
 
+namespace PP { 
+
+// data structure that represents an intron profile for a block and the inter-block section before the block
+// Author: Lars Gabriel
+class IntronProfile {
+    private:        
+        std::map < pair< int, int >, int > mapIntraBlock;
+        std::map < int, int > mapInterBlock;
+        int noSeq = 0;
+        double interBlock_mu = -1;
+        int blockNo, interBlockDistPos;        
+        std::vector<double> interBlockDist;
```

Comment:
At the moment  interBlockDistPos and interBlockDist serve no purpose and I have removed them. They were intended for pre-processing the Poisson distribution. The preprocessing is not yet implemented, should I add it now? I'm not sure because we might choose a different model or distribution for the introns in the inter-block regions.

Reply:
No, don't do this now. We will have bigger fish to fry - more important tasks. Yes, leave inactive code out. You can use git stash or simply copy the code elsewhere and take a note.